File: TransCoder_tokenized_test_set_functions/python/FIND_POSSIBLE_ROTATE_PAGE_ANGLE_NOT.py
def possibleOrNot ( a1, a2, b1, b2, c1, c2 ) :
    dis1 = ( pow ( b1 - a1, 2 ) + pow ( b2 - a2, 2 ) )
    dis2 = ( pow ( c1 - b1, 2 ) + pow ( c2 - b2, 2 ) )
    if ( dis1 != dis2 ) :
        print ( "No" )
    elif ( b1 == ( ( a1 + c1 ) //2.0)andb2==((a2+c2)//2.0)):
        print("No")
    else:
        print("Yes")
    

File: TransCoder_tokenized_test_set_functions/cpp/FIND_POSSIBLE_ROTATE_PAGE_ANGLE_NOT.cpp
void possibleOrNot ( long long a1, long long a2, long long b1, long long b2, long long c1, long long c2 ) {
  long long dis1 = pow ( b1 - a1, 2 ) + pow ( b2 - a2, 2 );
  long long dis2 = pow ( c1 - b1, 2 ) + pow ( c2 - b2, 2 );
  if ( dis1 != dis2 ) cout << "No";
  else if ( b1 == ( ( a1 + c1 ) / 2.0 ) && b2 == ( ( a2 + c2 ) / 2.0 ) ) cout << "No";
  else cout << "Yes";
}